src/runtime/driver/handle.rs

```diff
@@ -0,0 +1,272 @@
+use io_uring::squeue;
```

Comment:
Is there a comment block somewhere describing the design? Both for reviewers now but more importantly in the long run for readers and developers in the future?

Reply:
Hmm that is a good idea. I'll add it in the Handle file.

Reply:
Done.